client/my-sites/pages/page.jsx

```diff
@@ -411,12 +418,17 @@ const Page = React.createClass( {
 
 export default connect(
 	( state, props ) => {
+		const site = getSite( state, props.page.site_ID ) || {};
```

Comment:
By defaulting to the empty object here we lose the possibility of relying on `site`'s truthiness. Case in point, a few lines above:

```js
<Page key={ 'page-' + page.global_ID } page={ page } multisite={ ! this.props.site } />
```

`multisite` will never eval to true.

We should let `getSite` return `null` if need be, and treat `site` accordingly (i.e., test for truthiness before accessing properties — but `lodash#get` is our friend anyway).